docs/ARCHITECTURE.md

```diff
@@ -50,6 +50,27 @@ service manager.
 
 BPF helpers written in C and used by PID 1 can be found under `src/core/bpf/`.
 
+### Implementing Directives
```

Comment:
This is about units, not the global service manager config, hence maybe say "Implementing Unit Settings"

Reply:
(We so far called these things "settings", usually, but I think almost nowhere "directives")

Reply:
The public documentation calls an individual settings 'directive' in many many places, for example the full list: https://www.freedesktop.org/software/systemd/man/systemd.directives.html

Reply:
'directive' sounds more generic than 'setting' for me, including syntax of command line tools or so. So, I prefer 'settings' here and all other places, as this section only explains about unit file settings and D-Bus interfaces like `systemd-run -p`.

Reply:
ok, updated